src/pages/FavouriteMovie.js

```diff
@@ -55,10 +65,10 @@ class FavouriteMovie extends Component {
           </button>
         </div>
 
-        {this.state.movie_data.map((i) => (
-          <div className="div1">
-            <h4 className="par">{i.Title}</h4>
-            <img src={i.Poster} id="imageMovie"></img>
+        {this.state.movie_data.map((i, x1, x2) => (
```

Comment:
Please change var name from `i` to `movie` and change `x1` to `index`
What does `x2` represent?